benchmarks/perf-tool/release-configs/lucene-hnsw/index.json

```diff
@@ -0,0 +1,27 @@
+{
+  "settings": {
+    "index": {
+      "knn": true,
+      "number_of_shards": 24,
+      "number_of_replicas": 1,
+      "knn.algo_param.ef_search": 256
```

Comment:
nit: this line is redundant, ef_search is ignored for lucene and value of _k_ is used instead

Reply:
done